docs/docs/tutorials/run-notebooks-as-a-job.md

```diff
@@ -64,7 +64,24 @@ As mentioned above, the notebook job will run as an Argo-Workflows workflow. Thi
 :::
 
 :::note
-Only users in the `admin` or `developer` groups will have access to create notebook jobs. For more information regarding users and groups, please visit [How to configure Keycloak](/how-tos/configure-keycloak-howto.md)
+Only users in the `admin` or `developer` groups will have access to create notebook jobs. For more information regarding users and groups, please visit [How to configure Keycloak](/how-tos/configure-keycloak-howto.md).
+
+If your user is not part of one of the mentioned groups, you might see an error
+like this one when clicking on the job details:
+
+`Server returned status code 403 with message: 'workflows.argoproj.io is forbidden: User "system:serviceaccount:dev:argo-viewer" cannot create resource "workflows" in API group "argoproj.io" in the namespace "dev"`
+
+<div align="center">
+  <img src="/img/tutorials/jupyter-scheduler-permission-error.png" alt="Jupyter-Scheduler UI - permission error." width="100%"/>
+</div>
```

Comment:
This image pretty much duplicates the text above, but I think it's nice to have both here. The textual error is good for search and the image is good for figuring out where to look or how it should look in the UI.